kolibri/logger/serializers.py

```diff
@@ -49,11 +59,12 @@ class Meta:
 
     def get_pastattempts(self, obj):
         # will return a list of the latest 10 correct and hint_taken fields for each attempt.
-        return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).values('correct', 'hinted').order_by('-start_timestamp')[:10]
+        return AttemptLog.objects.filter(masterylog__summarylog=obj.summarylog).values('correct', 'hinted', 'error').order_by('-start_timestamp')[:10]
```

Comment:
'error' field doesn't exist on the current AttemptLog model

Reply:
Thanks, yeah, this doesn't exist in 0.9.